docs/components/buttons.md

```diff
@@ -44,12 +44,13 @@ API and source code:
 * `FlatButton`
     * [Class description](https://api.flutter.dev/flutter/material/FlatButton-class.html)
     * [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/flat_button.dart)
+    * [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/buttons/flat&gh_ref=develop)
```

Comment:
Playing with the dartpad demo, I realize that the buttons have their highlight color, which is not part of the M2 spec (remnant of M1), This should be probably be removed from all demos by setting highlight color to transparent

Reply:
Done